module/Application/view/partial/main-nav.phtml

```diff
@@ -39,14 +39,18 @@ $lang = $this->plugin('translate')->getTranslator()->getLocale();
                     <ul class="dropdown-menu language <?= ($lang == 'en') ? 'reverse' : '' ?> ">
                         <li>
                             <a class="<?= ($lang == 'en') ? 'disabled' : '' ?>"
-                               href="<?= $this->url('lang', ['lang' => 'en']) ?>">
+                               href="<?= $this->url('lang', ['lang' => 'en']) ?>"
+                               hreflang='en'
+                            >
```

Comment:
I think this is incorrect usage of `hreflang`, it is supposed to go on a `link` and not an `a` (and it should not be used on pages which redirect).

Reply:
It is proper usage according to this page: https://www.w3schools.com/TAGs/att_a_hreflang.asp

Reply:
Interesting, didn't know that. All good then!